### PR TITLE
Fix abnormal errorMessage (#114)

### PR DESCRIPTION
**Pull request by reopold**
The function **toString()** of errorBody does not return the response text.
It returns something like "okhttp3.ResponseBody$1@7fe86edf"
The function **string()** returns the response text.

I suggest the following algorithm: 
 - if errorBody contains some text, then return it, 
 - else return code and message from server (like "404 Not found").

### CODE REVIEW

File: telegram/src/main/kotlin/com/github/kotlintelegrambot/Bot.kt [hunk 1]
```diff
@@ -147,7 +147,10 @@ class Bot private constructor(
             false, null -> {
                 val errorMessage: String = when {
                     error?.message != null -> error.message!!
-                    response?.errorBody() != null -> response.errorBody().toString()
+                    (response?.errorBody()?.contentLength() ?: 0L) != 0L ->
+                        response?.errorBody()!!.string()
+                    response?.raw()?.message() != null ->
+                        "${response.raw().code()} ${response.raw().message()}"
```
Comment (vjgarciag96) [inline telegram/src/main/kotlin/com/github/kotlintelegrambot/Bot.kt:153]: I was thinking how to make this a bit more readable. What do you think about something like the next?:

```kotlin
            false, null -> {
                fun dispatchableError(message: String) =
                    listOf<TelegramError>(RetrieveUpdatesError(message))
                
                val errorMessage = error?.message
                if (errorMessage != null) {
                    return dispatchableError(errorMessage)
                }

                val errorBody = response?.errorBody()?.string()
                if (errorBody != null && errorBody.isNotEmptyAndNotBlank()) {
                    return dispatchableError(errorBody)
                }

                val rawHttpResponse = response?.raw()
                if (rawHttpResponse != null && rawHttpResponse.message().isNotEmptyAndNotBlank()) {
                    return dispatchableError(
                        "${rawHttpResponse.code()} - ${rawHttpResponse.message()}"
                    )
                }

                return dispatchableError(
                    "There was a problem retrieving updates from Telegram server"
                )
            }
```

**Review by vjgarciag96 — COMMENTED** [review]
Thank you so much for the PR @reopold! I added a little comment - once resolved I'll be happy to merge this

File: telegram/src/main/kotlin/com/github/kotlintelegrambot/Bot.kt (hunk 1 above)
Comment (reopold) [reply telegram/src/main/kotlin/com/github/kotlintelegrambot/Bot.kt:153]: Yes, in this form, readability is much better.

**Review by vjgarciag96 — APPROVED** [review]
LGTM!

**Pull request merged**